web_ui/frontend/app/api/docs/pelican-swagger.yaml

```diff
@@ -547,6 +624,11 @@ paths:
           required: true
           schema:
             $ref: "#/definitions/NamespaceForRegistration"
+        - in: header
+          name: X-CSRF-Token
+          description: The CSRF token for protecting Cross-Site Request Forgery (CSRF) attack. You will get this token by requesting `/api/v1.0/auth/whoami` and read response header `X-CSRF-Token`
```

Comment:
The CSRF token for protecting against Cross-Site Request Forgery (CSRF) attacks. Obtained by requesting `/api/v1.0/auth/whoami` and reading response header `X-CSRF-Token`